src/map/map.js

```diff
@@ -1,21 +1,24 @@
 import api from '../services/api.js';
+import createQuestLink from './create-quest-link.js';
+import loadProfile from '../services/load-profile.js';
+// import hasCompletedAllQuests from './hasCompletedAllQuests.js';
 
-//reference needed DOM elements
-
-const character = document.getElementById('character');
-const avatar = document.getElementById('avatar');
-const puppers = document.getElementById('puppers');
-const gusNbru = document.getElementById('gusNbru');
-
-//initialle: load profile with user
+loadProfile();
+const quests = api.getQuests();
 const user = api.getUser();
 
 if(!user) {
     window.location = './';
 }
 
-//copy data from object to DOM properties
-character.textContent = character.name;
-avatar.src = '../public/assets/avatars/' + user.character + '.png';
-puppers.textContent = user.puppers;
-gusNbru.textContent = user.gusNbru;
\ No newline at end of file
+const div = document.getElementById('barn');
```

Comment:
You need to double-check the id/class on this div because it's coming back as null which is why your links aren't loading.